framework/bootstrap/configcontroller.go

```diff
@@ -252,6 +252,10 @@ func NewConfigController(config *bootconfig.Config, cfg *rest.Config) (ConfigCon
 
 	epsToIstioResHandler := func(old resource.Config, cfg resource.Config, e Event) {
 		ep := cfg.Spec.(*v1.Endpoints)
+		// some special endpoints have no subsets, like ep/k8s.io-minikube-hostpath in minikube, ignore
+		if ep.Subsets == nil {
```

Comment:
check-for-nil-subsets is someshow strange. we can refer to logic in istio:

```go
	hostname := kube.ServiceHostname(endpoints.Name, endpoints.Namespace, c.domainSuffix)
	c.RLock()
	svc := c.servicesMap[hostname]
	c.RUnlock()

	if svc != nil {
```

Reply:
fix in the future